02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>This is more of a question, since now in pandas you can have pyarrow dtypes, can you read or cast the "object" typo as a pyarrow dtype? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Hmm, I don't think we've really dug into handling the pyarrow-backed pandas dataframes.</p><p> </p>